THIRD-PARTY-NOTICES.TXT

```diff
@@ -57,3 +57,20 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+License notice for OpenTelemetry .NET (https://github.com/open-telemetry/opentelemetry-dotnet)
```

Comment:
note that the next one (StackExchange.Redis) comes from a different repo: https://github.com/open-telemetry/opentelemetry-dotnet-contrib. We may need to update this when we pull that code in.

Reply:
```suggestion
License notice for OpenTelemetry
```